flake.nix

```diff
@@ -38,7 +38,7 @@
           src = ./razer_control_gui;
 
           postConfigure = ''
-            substituteInPlace src/device.rs --replace '/usr/share/razercontrol/laptops.json' '${./razer_control_gui/data/devices/laptops.json}'
+            substituteInPlace src/daemon/device.rs --replace '/usr/share/razercontrol/laptops.json' '${./razer_control_gui/data/devices/laptops.json}'
```

Comment:
Its needed (from what I understand) because every package in nix has its own filestructure in the "store" and cant access files outside of it local paths.

Reply:
That is what I was thinking. I will remove the need for that by trying to read the first path, and if it fails, the second path.